index.js

```diff
@@ -69,10 +87,30 @@ async function run () {
   return '🏁⛑'
 }
 
-async function getTeamPingIssues (octokit, org, team, authors, commenters, since = '2019-01-01', projectBoard, ignoreRepos, ignoreLabels) {
+async function getTeamMentionsIssues (octokit, org, team, authors, commenters, since, projectBoard, ignoreRepos, ignoreLabels) {
   // Search for open issues in repositories owned by `org`
   // and includes a team mention to `team`
   let query = `per_page=100&q=is%3Aopen+org%3A${org}+team%3A${team}`
+
+  query = query.concat(await buildExceptions(authors, commenters, since, projectBoard, ignoreRepos, ignoreLabels))
+
+  console.log(`🔎 Searh query 🔎 ${query}`)
+  return await octokit.request(`GET /search/issues?${query}`)
+}
+
+async function getTeamReviewRequests (octokit, org, team, authors, commenters, since, projectBoard, ignoreRepos, ignoreLabels) {
```

Comment:
Agreed, I think we want to have PR's from team members added to the queue to be reviewed. So keeping the same ignore lists doesn't seem to make since.